test/transform.globalize.js

```diff
@@ -5,6 +5,8 @@ var mocha = require('mocha');
 var transform = require('../src/transform/globalize');
 
 mocha.describe('transform/globalize', function () {
+  var generatedModuleName = '__1dd241c4cd3fd1dd89c570cee98b79dd';
```

Comment:
I'm not sure what part the test data generates this particular module name, but it would be nice to pull that particular string out to this same scope so that the different `mocha.describe(…)` blocks don't go out of sync.